packages/astro/src/core/errors/errors-data.ts

```diff
@@ -748,27 +748,6 @@ export const ResponseSentError = {
 	message: 'The response has already been sent to the browser and cannot be altered.',
 } satisfies ErrorData;
 
-/**
- * @docs
- * @description
```

Comment:
Irrelevant of if this should be merged or not (agree with Matthew's comment), errors should never be deleted (at least, not in patches or minors), they should be deprecated using the `@deprecated` JSDoc tag

Reply:
Oh right, it's also used for docs.